### PR TITLE
Truncate link titles

### PR DESCRIPTION
**Pull request by neanias**
Otherwise we're going to end up with another rick-rolling occasion

### CODE REVIEW

File: ircbot/plugins/links.py [hunk 1]
```diff
@@ -22,6 +22,14 @@ def allowed_to_process(bot, channel):
     return True
 
 
+def truncate_title(title):
+    title = title.replace("\n", "")
+    if len(title) > 120:
```
Comment (hreeder) [inline ircbot/plugins/links.py]: Please move this out to a config file value, probably `config['IRC']['link_max_len']`

File: ircbot/plugins/links.py [hunk 2]
```diff
@@ -36,8 +44,7 @@ def link_title_parse_hook(bot, channel, sender, message):
                     return
 
                 soup = BeautifulSoup(r.text, 'html.parser')
-                title = soup.head.title.text.strip()
-                title = title.replace("\n", "")
+                title = truncate_title(soup.head.title.text.strip())
```
Comment (hreeder) [inline ircbot/plugins/links.py]: Can we still leave this as two lines - one doing `soup.head.title.text.strip()` and the other doing `title = truncate_title(title)` - I feel it's a bit more readable?

**Review by hreeder — CHANGES_REQUESTED** [review]

File: ircbot/plugins/links.py (hunk 1 above)
Comment (hreeder) [reply ircbot/plugins/links.py]: Ok, just saw commit db3602a - I prefer that config file location, nice one :+1:

**Comment by neanias** [comment]
@hreeder changes made

**Review by hreeder — APPROVED** [review]

**Comment by hreeder** [comment]
LGTM :+1:

**Pull request merged**